src/test/java/uk/gov/ons/ctp/response/casesvc/service/CaseGroupServiceTest.java

```diff
@@ -64,6 +65,8 @@ public void givenCaseGroupStatusWhenCaseGroupStatusTransitionedThenTransitionIsS
     caseGroupService.transitionCaseGroupStatus(caseGroup, categoryName, caseGroup.getPartyId());
 
     // Then
+    Assert.assertNotNull(caseGroup.getStatusChangeTimestamp());
```

Comment:
Maybe change the name of this test to not be so rubbish, I know you didn't do it originally